tox.ini

```diff
@@ -46,117 +70,108 @@ setenv =
 # This version should be kept in sync with the one declared in
 # tools/pinning/oldest/pyproject.toml.
 basepython = python3.7
-commands =
-    {[testenv]commands}
-setenv =
-    {[testenv]setenv}
-    CERTBOT_OLDEST=1
-
-[testenv:acme-oldest]
-basepython =
-    {[testenv:oldest]basepython}
-commands =
-    {[base]install_and_test} acme[test]
-setenv =
-    {[testenv:oldest]setenv}
-
-[testenv:apache-oldest]
-basepython =
-    {[testenv:oldest]basepython}
-commands =
-    {[base]pip_install} acme[test] certbot[test] certbot-apache
-    pytest certbot-apache
-setenv =
-    {[testenv:oldest]setenv}
-
-[testenv:apache-v2-oldest]
-basepython =
-    {[testenv:oldest]basepython}
-commands =
-    {[base]pip_install} acme[test] certbot[test] certbot-apache[dev]
-    pytest certbot-apache
-setenv =
-    {[testenv:oldest]setenv}
+setenv = CERTBOT_OLDEST=1
+commands = {[testenv:py]commands}
 
-[testenv:certbot-oldest]
-basepython =
-    {[testenv:oldest]basepython}
-commands =
-    {[base]pip_install} acme[test] certbot[test]
-    pytest certbot
-setenv =
-    {[testenv:oldest]setenv}
+[testenv:cover{,-win,-posix}]
```

Comment:
Can we do away with the `-win`, `-posix` suffixes here?

Reply:
I don't think we can get rid of `-win,-posix` in either case. I initially found this behavior confusing and it looks like you might too, but this test environment is inheriting settings from the default `testenv`. See https://tox.wiki/en/latest/user_guide.html#test-environments. Because of that `-win` allows us to skip installing certbot-apache which doesn't work on Windows and `-posix` allows us to have `cover-{win,posix}` in our default list of tox environments, running one, and skipping the other depending on whether `tox` is run on posix or windows.

Reply:
Ah, that makes sense. Okay!